pkgs/development/python-modules/python-zbar/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, python
+, pillow
+, zbar }:
+
+buildPythonPackage rec {
+  pname = "python-zbar";
+  version = "0.23.90";
+
+  src = fetchFromGitHub {
+    owner = "mchehab";
+    repo = "zbar";
+    rev = version;
+    sha256 = "sha256-FvV7TMc4JbOiRjWLka0IhtpGGqGm5fis7h870OmJw2U=";
+  };
+
+  propagatedBuildInputs = [ pillow ];
+
+  buildInputs = [ zbar ];
+
+  preBuild = ''
+    cd python
+  '';
+
+  pythonImportsCheck = [ "zbar" ];
+
+  # Three of the tests are failing
+  doCheck = false;
```

Comment:
Why do we not disable those 3?

Reply:
The reason I disabled the checks is because the tests are contained at this [single file](https://github.com/mchehab/zbar/blob/master/python/test/test_zbar.py). It is possible to call only certain tests from it?

Reply:
I ran it again with the checks and it seems the tests are now running fully with no issues. I removed the line.